site/src/docs/components/header/index.mdx

```diff
@@ -64,6 +64,12 @@ Coming soon!
 
 Coming soon!
 
+### With navigation language selector
```

Comment:
Navigation language selector is sort of a sibling component with UniversalBar and others, so this should be of same heading level. So this should be with an extra hashtag: `#### With navigation language selector`